templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -189,16 +189,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Second and third sentence here are redundant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>